FirebaseInAppMessaging/Sources/Analytics/FIRIAMAnalyticsEventLoggerImpl.m

```diff
@@ -15,7 +15,7 @@
  */
 
 #import <TargetConditionals.h>
-#if TARGET_OS_IOS
+#if TARGET_OS_IOS || TARGET_OS_TV
```

Comment:
Nit: the end of the file has `#endif  // TARGET_OS_IOS` can you also include the TV conditional too?

Reply:
Resolved in 1307336.